charts/drax/templates/configmap-loki-tenant-override-config.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: loki-tenant-override-config
+  namespace: default
+data:
+  tenant_override_config.yaml: |
+    overrides:
+      "notifications":
+        # Sets the retention period for notifications to 672 hours (28 days).
+        retention_period: 24h
+      "logs":
+        # Sets the retention period for logs to 672 hours (28 days).
+        retention_period: 672h
```

Maybe also remove the comment, or put it on the same line to prevent as much as possible the same thing as for the notifications:

```suggestion
        retention_period: 672h  # 28 days
```